embassy-nrf/src/lib.rs

```diff
@@ -162,7 +163,10 @@ pub fn init(config: config::Config) -> Peripherals {
     while r.events_lfclkstarted.read().bits() == 0 {}
 
     // Init GPIOTE
-    crate::gpiote::init(config.gpiote_interrupt_priority);
+    gpiote::init(config.gpiote_interrupt_priority);
+
+    // init RTC time driver
+    time_driver::init();
```

Comment:
Configurable irq priority would be nice to be consistent with `gpiote::init()`

Reply:
Agreed! Will do this on a follow-up PR, since the previous code couldn't do this anyway (when using `embassy::main`). There's an issue for this already (#235)